apps/sr-frontend/src/app/App.tsx

```diff
@@ -27,6 +29,15 @@ export default function App() {
         <DatabaseProvider>
           <CharacterProvider>
             <CalcProvider>
+              <CssBaseline />
```

Comment:
Bad merge, this stuff shouldn't be here anymore (Sorry i changed it out from underneath you)
fix should be to delete all your changes here, then add LightConeInventory to PageHome.tsx
Perhaps wait for https://github.com/frzyc/genshin-optimizer/pull/1881 to merge first, as that will end up causing a conflict anyways for you